src/Web/Views/Shared/_Layout.cshtml

```diff
@@ -60,19 +62,26 @@
     <header class="navbar navbar-light navbar-static-top">
         <div class="container">
             <article class="row">
-                <section class="col-lg-6 col-md-5 col-xs-11">
+                <section class="col-lg-2 col-md-4 col-xs-11">
                     <a asp-area="" asp-controller="Catalog" asp-action="Index" class="navbar-brand">
                         <img src="~/images/brand.png" alt="Microsoft Fraud Protection Merchant Application" title="@apiBaseUrl" />
                     </a>
                 </section>
                 <partial name="_LoginPartial" />
+                @using (Html.BeginForm("Update", "Environment", FormMethod.Post, new { id = "environmentupdate" }))    
+                {
+                    <div class="form-group">
+                        <label for="env">Environment Id: </label>
+                        <input type="text" value="@data" id="endvironmentId" name="env" title="@data"/>
```

Comment:
why do you need to set 'title' if it's already the value too?

Reply:
for showing complete guid in textbox, we really need very big textbox so we have textbox and Title to show complete value if anyone hover on it.